src/app/portfolio/portfolio.component.html

```diff
@@ -45,7 +47,7 @@
                 </div>
               </div>
               <div class="text-3xl mt-4 lg:text-4xl tracking-wider font-light uppercase">
-                {{ item.campaign.name }}
+                <a routerLink="{{ '/offers/' + item.campaign.ansName | issuerPath }}">{{ item.campaign.name }}</a>
```

Comment:
```suggestion
                <a routerLink="{{ '/offers/' + item.campaign.ansName | issuerPath }}">
                  {{ item.campaign.name }}
                </a>
```

If something is a title or any important stuff on the screen, try not to put it on the far right side of the screen. This messes up readability. If something can be a one-liner, it doesn't mean it should be. Sometimes, it also happens to me, but I found out that it is a clearer approach when trying to find something inside the code. 😉